### PR TITLE
add media helper to create a new media and add file to an existing one

### PR DESCRIPTION
**Pull request by aureldent**

### CODE REVIEW

File: lumapps/api/helpers/medias.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+def add_new_media_file_of_given_lang_to_existing_media(
+    client: ApiClient,
+    media: dict,
+    file_data_or_path: Union[bytes, str],
+    filename: str,
+    mimetype: str,
+    lang: str = "en",
+    croppedContent: bool = False,
+) -> Union[dict, None]:
+    """[summary]
```
Comment (maximevast) [inline lumapps/api/helpers/medias.py]: missing summary

File: lumapps/api/helpers/medias.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+    Raises:
+        Exception: The data or file path type provided is not supported.
+
+    Returns:
+        Union[dict, None]: Return the uploaded media if successfull, None otherwise.
+    """
+
+    if isinstance(file_data_or_path, str):
+        file_data = open(file_data_or_path, "rb")
```
Comment (ulodciv) [inline lumapps/api/helpers/medias.py]: What about pathlib support?

File: lumapps/api/helpers/medias.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+        files=files_tuple_list,
+    )
+    doc = response.json().get("items")
+
+    if doc:
+        return doc[0]
+    return None
+
+
+def add_new_media_file_of_given_lang_to_existing_media(
```
Comment (ulodciv) [inline lumapps/api/helpers/medias.py]: Perhaps a shorter name like add_lang_media_file or add_media_file_for_lang?

File: lumapps/api/helpers/medias.py [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+    Keyword Arguments:
+        lang (str): The lang of the file to upload (default: "en").
+        prepare_for_lumapps (bool): (default: True)
+
+    Raises:
+        Exception: The data or file path type provided is not supported.
+
+    Returns:
+        Union[dict, None]: Return the uploaded file if successfull, None otherwise.
```
Comment (ulodciv) [inline lumapps/api/helpers/medias.py]: Union[dict, None] -> Optional[None]

File: lumapps/api/helpers/medias.py [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+                        (eg: provider=<my_provider>/site=<my_site_id>/resource=<my_parent_folder_id>)
+        filename (str): The name of the file to upload. Once uploaded the file will appear with that name.
+        mimetype (str): The mimeType fo the file to upload.
+
+    Keyword Arguments:
+        lang (str): The lang of the file to upload (default: "en").
+        croppedContent (bool): Wether to add the file to the croppedContent instead or content (default: False)
+
+    Returns:
+        Union[Dict, None]: The updated media if succesfull, otherwise None.
```
Comment (ulodciv) [inline lumapps/api/helpers/medias.py]: Dict -> dict

File: lumapps/api/helpers/medias.py (hunk 2 above)
Comment (aureldent) [reply lumapps/api/helpers/medias.py]: No python2 compatible, surely when we gave up on python2 suport !

**Review by ulodciv — APPROVED** [review]

**Pull request merged**